pytest.ini

```diff
@@ -0,0 +1,11 @@
+[pytest]
+testpaths = tests/integration
+python_files = test_*.py
+python_functions = test_*
+markers =
+    unit: marks tests as unit tests
+    integration: marks tests as integration tests
+addopts = -v -ra -q
```

Comment:
`-v` and `-q` conflict:

-v is for verbose output
-q is for quiet output

Using both together is contradictory, and -q will override -v